### PR TITLE
Fix creditmemo tax calculation bug when using multiple discounts (again)

### PR DESCRIPTION
**Pull request by seansan**
Taken from #88 / maybe can be edited now

### CODE REVIEW

**Comment by seansan** [comment]
@tmotyl Can you access now? It has been a while since I made these changes. Also need to dive in again. This is why I also asked for confirmation from other testers/people that work with credit memo

**Comment by tmotyl** [comment]
yes, it's editable now

**Comment by sreichel** [comment]
From #88 

> When adding several product types and utilizing several (more than 1) discount code the credit memo calculation is not correct.

Please correct me, but magento does not support multiple discount codes by default.

File: app/code/core/Mage/Sales/Model/Order/Creditmemo/Total/Subtotal.php [hunk 1]
```diff
@@ -58,8 +58,8 @@ public function collect(Mage_Sales_Model_Order_Creditmemo $creditmemo)
         $creditmemo->setSubtotalInclTax($subtotalInclTax );
         $creditmemo->setBaseSubtotalInclTax($baseSubtotalInclTax);
 
-        $creditmemo->setGrandTotal($creditmemo->getGrandTotal() + $subtotal);
-        $creditmemo->setBaseGrandTotal($creditmemo->getBaseGrandTotal() + $baseSubtotal);
+        $creditmemo->setGrandTotal($creditmemo->getGrandTotal() + $subtotalInclTax);
+        $creditmemo->setBaseGrandTotal($creditmemo->getBaseGrandTotal() + $baseSubtotalInclTax);
```
Comment (sreichel) [inline app/code/core/Mage/Sales/Model/Order/Creditmemo/Total/Subtotal.php]: Is this tested with different tax settings?

**Comment by LeeSaferite** [comment]
@sreichel You are correct, Magento does not support multiple coupon codes.

**Comment by sreichel** [comment]
So it should be fixed in 3rd-party code?

**Pull request closed without merging**